data/competencies.yml

```diff
@@ -312,6 +312,19 @@
 
 # Mid to Senior 1, Technical
 
+- id: 272be0d0-6b55-11e9-ae4e-0583f673f115
+  summary: Builds products ensuring they take adequate steps to protect user data
```

Comment:
Does there need to be something about making sure junior devs to do so too? If not at this level, then the level above.